chart/values.yaml

```diff
@@ -357,6 +357,11 @@ components:
     # This can be generated with: openssl dhparam 4096 2> /dev/null | base64 -w 0
     # http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_dhparam
     sslDHParam:
+    # enabled cronjob to restart the proxy deployment
+    restarter:
+      enabled: true
```

Comment:
Can we disable this by default? I don't think it should be on by default in GP-SH installations.

Reply:
Is now disabled by default and set as enabled in the `certmanager` Terraform module.